pycode/test/test_epidata_get_simulation_data.py

```diff
@@ -48,29 +49,48 @@ def setUp(self):
-
-        arg_dict_all = {"read_data": False, "file_format": "json", "out_folder": self.path, "no_raw": False}
-
-        arg_dict_rki = {**arg_dict_all, "make_plot": dd.defaultDict['make_plot'],
-                        "fill_dates": dd.defaultDict['fill_dates'], "moving_average": dd.defaultDict['moving_average'],
-                        "split_berlin": dd.defaultDict['split_berlin']}
-
-        arg_dict_divi = {**arg_dict_all, "end_date": dd.defaultDict['end_date'],
-                         "start_date": dd.defaultDict['start_date'], "update_data": dd.defaultDict['update_data']}
-
-        mock_vaccine.assert_called()
-        mock_vaccine.assert_called_with(**arg_dict_all)
+    @patch('epidemiology.epidata.getVaccinationData.get_vaccination_data')
+    def test_get_call_sub_functions(self, mock_vaccination, mock_agep,
+                                    mock_popul, mock_rki, mock_divi):
+
+        [read_data, file_format, out_folder, no_raw, end_date, impute_dates,
+         make_plot, moving_average, split_berlin, start_date] = [False,
+                                                                 "json_timeasstring", os.path.join(dd.defaultDict['out_folder']),
+                                                                 False, dd.defaultDict['end_date'],
+                                                                 dd.defaultDict['impute_dates'],
+                                                                 dd.defaultDict['make_plot'],
+                                                                 dd.defaultDict['moving_average'],
+                                                                 dd.defaultDict['split_berlin'],
+                                                                 dd.defaultDict['start_date']]
+
+        gsd.get_simulation_data(
+            read_data, file_format, out_folder, no_raw, end_date, impute_dates,
+            make_plot, moving_average, split_berlin, start_date)
+
+        arg_dict_all = {
+            "read_data": dd.defaultDict['read_data'],
+            "file_format": dd.defaultDict['file_format'],
+            "out_folder": os.path.join(dd.defaultDict['out_folder']),
+            'no_raw': dd.defaultDict["no_raw"]}
+
+        arg_dict_rki = {
+            **arg_dict_all, "make_plot": dd.defaultDict['make_plot'],
+            "impute_dates": dd.defaultDict['impute_dates'],
+            "moving_average": dd.defaultDict['moving_average'],
+            "split_berlin": dd.defaultDict['split_berlin']}
+
+        arg_dict_divi = {
+            **arg_dict_all, "end_date": dd.defaultDict['end_date'],
+            "start_date": dd.defaultDict['start_date'],
+            "moving_average": dd.defaultDict['moving_average']}
+
+        arg_dict_vaccination = {
+            **arg_dict_all,
+            "make_plot": dd.defaultDict['make_plot'],
+            "moving_average": dd.defaultDict['moving_average']}
+
+        mock_vaccination.assert_called()
```

Comment:
Please try to maintain a certain consistency, e.g., the order of calls of the different functions (population, rki, divi, vaccination)